src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -0,0 +1,170 @@
+        for commit in commits:
+            repo: Repository = Repository.objects.get(id=commit.repository_id)
+            # Hardcoded to only accept getsentry/sentry repo for now
+            if repo.external_id == "getsentry/sentry":
+                base_commit = commit
+                break
+
+        if not base_commit:
+            metadata["autofix"] = {
+                **metadata["autofix"],
+                "completedAt": datetime.now().isoformat(),
+                "status": "ERROR",
+                "fix": None,
+                # Hardcoded to only accept getsentry/sentry repo for now
+                "errorMessage": "No valid base commit found for release; only getsentry/sentry repo is supported right now.",
+            }
+            group.data["metadata"] = metadata
+            group.save()
+
+            return HttpResponse(
+                status=400,
+            )
+
+        try:
+            seer_connection_pool.urlopen(
+                "POST",
+                "/v0/automation/autofix",
+                body=json.dumps(
+                    {
+                        "base_commit_sha": base_commit.key,
+                        "issue": {
+                            "id": str(group.id),
+                            "title": group.title,
+                            "events": [event_stacktrace],
+                        },
+                        "additional_context": data.get("additional_context", ""),
+                    }
+                ),
+                headers={"content-type": "application/json;charset=utf-8"},
+            )
+
+            # Mark the task as completed after TIMEOUT_SECONDS
+            check_for_timeout.apply_async(
+                kwargs={
+                    "group_id": group.id,
+                    "created_at": created_at,
+                },
+                countdown=TIMEOUT_SECONDS,
+            )
+        except Exception as e:
+            metadata["autofix"] = {
+                **metadata["autofix"],
+                "completedAt": datetime.now().isoformat(),
+                "status": "ERROR",
+                "fix": None,
+                "errorMessage": "Failed to send autofix to seer.",
+            }
+
+            logger.exception(
+                "Failed to send autofix to seer",
+                extra={
+                    "group_id": group.id,
+                    "created_at": created_at,
+                    "exception": e,
+                },
+            )
+
+        group.data["metadata"] = metadata
+
+        group.save()
+
+        return HttpResponse(
+            status=200,
+        )
+
+    def get(self, request: Request, group: Group) -> HttpResponse:
+        metadata = group.data.get("metadata", {})
+        autofix_data = metadata.get("autofix", None)
+
+        return HttpResponse(
+            status=200,
+            content_type="application/json",
+            content=json.dumps(autofix_data) if autofix_data else None,
+        )
```

Comment:
How can you tell apart no autofix v no autofix _yet_ (because it's still processing)?

Reply:
Autofix is set in the metadata on the request, if it's still processing the status is set as `PROCESSING`

https://github.com/getsentry/sentry/blob/6dffc8bc4ee9db8723ba186db0a98ac6d9be3e3d/src/sentry/api/endpoints/group_ai_autofix.py#L59-L62